ver/us/splat.yaml

```diff
@@ -369,7 +369,12 @@ segments:
       - [0x511E0, vtx, theater/wall_shadows_vtx]
       - [0x51260]
       - [0x51540, ia8, ui/no_controller, 128, 32]
-    - [auto, .data, crash_screen]
+    - start: auto
+      type: .data
+      name: crash_screen
+      subsegments:
+      - [0x52690, i1, crash_screen/font, 32, 63]
```

Comment:
the start addr here can be auto I'd think

Reply:
nm

Reply:
This gives `Error: crash_screen/font should end at 0x4FADC, but it ends at 0x5278C`